src-qt5/core-utils/lumina-config/i18n/lumina-config_sv.ts

```diff
@@ -1475,7 +1475,7 @@
     <message>
         <location filename="../pages/page_interface_desktop.ui" line="86"/>
         <source>Display Removable Media Icons</source>
-        <translation type="unfinished"></translation>
+        <translation>unfinished</translation>
```

Comment:
This needs to be finished, or reverted back to having the `type="unfinished"` tag.